README.md

```diff
@@ -1,17 +1,18 @@
 # Stargate
-An open source data gateway.
+
+#### An open source data gateway.
 
 [![stargate](http://img.youtube.com/vi/2ltVf2EscmM/0.jpg)](http://www.youtube.com/watch?v=2ltVf2EscmM "https://stargate.io/assets/images/explainer-cover.png")
 
 > Stargate is a data gateway deployed between client applications and a database. It's built with extensibility as a first-class citizen and makes it easy to use a database for any application workload by adding plugin support for new APIs, data types, and access methods.
 
-- For information about how to use Stargate, visit [stargate.io](https://stargate.io/)
+- For more information about how to use Stargate, visit [stargate.io](https://stargate.io/)
 
 **Discord bot using Stargate**
 
 ## Phase 1 Goals
 
-- Video tutorial 5-10min of using Stargate (serverless)
+- A video tutorial (ideally, 5-10 min) on how to use Stargate (serverless)
```

Comment:
I'm not sure if 5-10 mins _can_ cover the use of Stargate.. Could we use something like "giving an intro to Stargate?"

Reply:
Actually, that was something Eddie had planned. I'm not sure if we should change it, since he might lose the perspective of his plan.

Reply:
Yep, it will be an intro on CRUD